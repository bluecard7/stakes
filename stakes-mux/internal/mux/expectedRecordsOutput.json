{"records":[{"id":"081701cf-c807-4408-88d1-1d2eadec0380","clockIn":"2006-01-02T00:00:00Z","clockOut":"2006-01-02T08:00:00Z"}]}