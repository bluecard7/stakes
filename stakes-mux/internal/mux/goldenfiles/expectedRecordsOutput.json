{"records":[{"id":"00000000-0000-0000-0000-000000000001","clockIn":"2006-01-02T00:00:00Z","clockOut":"2006-01-02T08:00:00Z"}]}